docs/source/scientific_reference/1_time_series.rst

```diff
@@ -0,0 +1,165 @@
+.. _sciref_series:
+
+Scientific reference: time series
+=================================
+
+``sktime``'s conceptual model is based on a number of "time series" related mathematical
+objects:
+
+* "time series", observations of a single process made over time.
+Example: average temperature in London, on a given day.
+* "time series panels", collections of multiple time series, independently observed.
+Example: fever curves of 10 specific hospital patients, for the time in the hospital.
+* "hierarchical time series", collections of multiple time series where individual
```

Comment:
panel and hierarchical are commonly used terms. The intuition is usually the same, though the precise formal meaning that authors assign to it differs across sources.

Reply:
I'm not sure I understand the distinction of panel and hierarchical based purely on the explanation and example. Maybe you can quote some sources?

Reply:
a panel is just a collection, whereas a hierarchical time series has hierarchy labels attached to different series.
What complicates the conceptualization is that panels can be seen as a sub-case of hierarchical, where the hierarchy label is a (flat) set of categories or integer indices.###

Reply:
I don't really understand the difference between a multivariate instance and a hierarchical instance, and why that distinction is important. 

1. Is the latter is meant to include relationship data (e.g. spatial data)? If so, is there a mechanism for encoding this? A EEG is under this interpretation hierarchical, as is uman activity recognition, but afaik we do not have a standard for encoding this within the data structure, so the difference is not in the implementation. 
2. Or do we allow different sampling processes/start times for hierarchical? What actually is the "hierarchy"? If so, then it would have to be stored as a pd.Series, and that could be a valid distinction. 
3. Or are some series actually derived from others? Such as total sals = sum of all sales at the next level of hierarchy?